README.md

```diff
@@ -127,13 +127,73 @@ First-Party Sets is a proposal to standardize a mechanism that solves this issue
 
 # Applications
 
-In support of the various browser privacy models, first-party sets only control when embedded content that would otherwise be considered third-party can access its own state. Examples:
+Various applications of First-Party Sets have been suggested by different parties. The list below is an effort to compile suggested applications. 
+
+## Initial Proposed Applications
+
+**Bounce/navigation-based tracking protections**\
+_Interest from: [Safari](https://github.com/privacycg/first-party-sets/issues/53#issuecomment-902396090), Chrome, Edge_
+
+To allow for first-party cross-site single-sign-on flows to continue working, where [navigation-based tracking mitigations](https://privacycg.github.io/nav-tracking-mitigations/) are applied by browsers. Sign-on typically involves exchange of authentication tokens or other user identifiers via URL parameters on top-level navigation and redirects; which may get classified as navigational tracking.
+
+**Relaxing full third-party cookie blocking for cookies with the SameParty attribute**\
+_Interest from: Chrome, Edge_
+
+To help websites maintain various uses of cross-site cookies within the same organization such as for SSO, consent management, sandboxed domains, etc. 
+Cookies need to be labeled with the [SameParty attribute](#sameparty-cookies-and-first-party-sets).
+
+Prior art: Firefox and Edge use the [entity list](https://github.com/mozilla-services/shavar-prod-lists#entity-list), Safari’s [Quirks.cpp](https://webkit-search.igalia.com/webkit/rev/0be50a7e5c29546e4c5f27ee6b808f9aaadb9f80/Source/WebCore/page/Quirks.cpp#1154) and [affiliated domains](https://www.w3.org/2017/11/06-webappsec-minutes.html#item12) proposal.
```

Comment:
I think other Mozillians have stated this before: we don't consider this prior art. This is a workaround to more quickly get to a place where authority is fully derived from the information in the address bar.

(Safari naming it "Quirks" suggests a similar line of thinking, but I won't speak for them.)

Reply:
The intention here is to document similar relaxation of cross-site cookie access currently shipping in other major browsers, based on curated lists of related/affiliated sites. "Prior Art" happens to be a succinct term that explains that intention, but happy to use a different phrase. :)

Would "Similar workaround currently employed in other browsers" be more appropriate?

Reply:
Understood. We didn't mean for this to come off as us presuming Mozilla's support for FPS standardization. It was only referring to similar mechanisms that exist in browsers.

Reply:
While I don't read it as such, the message here is pretty clear: "FPS is the permanent solution".

What I think is missing from this is a discussion of the intent behind the mechanisms.  Mozilla has been clear that our use of the entity list is a stopgap.  The entity list is terrible for a range of reasons, some of which FPS aims to solve, some of which FPS also shares.

```suggestion
```

Reply:
@martinthomson @annevk - FYI, the question of whether FPS is intended as an interim/stopgap solution [came up](https://github.com/w3ctag/design-reviews/issues/342#issuecomment-1076595569) again today on the FPS TAG review thread. I wanted to surface [my response](https://github.com/w3ctag/design-reviews/issues/342#issuecomment-1076737573) here as well, and hear from you whether Firefox's use of the Disconnect entities list without a (yet to be announced) deprecation date might mean that you would support FPS as a medium-term solution to unblock comprehensive tracking protections in service of interoperability:

> [...] if other browsers would prefer to engage with FPS as a medium-term solution, I think that would still be a vast improvement over reliance on lists and heuristics. Blocklists are never exhaustive, they fail open, and are prone to errors such as those evidenced in the Disconnect entities list. Heuristics cause platform predictability issues.

Reply:
I think it's worth pointing out something else here. We use the list for blocking known trackers access to cookies and storage in Firefox (release version). This is to prevent our users from being tracked. That's a fundamentally different goal from FPS.

FPS is a Google proposal to augment state partitioning. We don't think state partitioning (which we're calling Total Cookie Protection and we haven't rolled out to release yet, but are actively working on) needs to be augmented with lists in that manner. We may or may not keep the tracker list around, but it fundamentally shouldn't change the privacy argument much, except perhaps when we block for fingerprinting or other aspects we don't have a holistic solution for yet.

We do have heuristics in place around TCP to mitigate breakage, but we believe we can reduce those over time.

So respectfully, please stop making this comparison.

Reply:
@annevk Thanks for your note, and the discussion on the call. The discussion was very educational for me, because it made me question my assumption that we were all targeting the same end-state regarding tracking protections. As Chrome has announced, our desired end-state is to **phase out** usage of third-party cookie completely, without use of tracker blocklists, and ideally even without temporary escape hatches like Storage Access API. 

Is comprehensive third-party cookie blocking/partitioning on **all cross-site contexts** something you consider as your desired end-state? The reason I'm asking is if you only plan to intervene on known trackers (for cross-site cookie access, navigational tracking, fingerprinting, and other kinds of tracking protections); that has significantly fewer compat implications than a comprehensive approach that intervenes on all cross-site contexts.

Reply:
> Is comprehensive third-party cookie blocking/partitioning on all cross-site contexts something you consider as your desired end-state? 

Yes.  

Partitioning of all site-level state would be ideal.  We want the promise we make to be very clear: if the address bar changes, that's a new space.  When we get there, only the address bar matters.

Your previous statements about blocklists is probably where we differ most.  You seem to view blocklists as inherently bad.  We recognize their limitations and failings, but I think our view is somewhat more pragmatic: there will be bad actors and we'll need some way to limit what those bad actors can do.  That might entail use of things like blocklists, for actors with established patterns of bad behaviour.  That might be true even when the technical protections we have a more robust.  As you know, our ability to prevent tracking or secure the browser is finite, so we're simply allowing for the possibility that those systems we've built for might still be useful.  And technical enforcement is tricky and inflexible, so there might be a place for some manual processes.

Reply:
Hey @martinthomson , super late to this, been learning a lot of the issues in this space from these and other threads. Hoping you can help me relate a couple of your previous points here to current state (both of RWS and the web more broadly):

>  Mozilla has been clear that our use of the entity list is a stopgap. The entity list is terrible for a range of reasons, some of which FPS aims to solve, some of which FPS also shares.

I think [this](https://github.com/mozilla-services/shavar-prod-lists?tab=readme-ov-file#disconnect-entitylistjson) is saying that Firefox is using the entities.json list to allow a "known tracker" (known according to other Disconnect lists) to be loaded if it's from a site that is owned by the top level site, according to that list. I'm a little unclear how this impacts Total Cookie Prevention but it does seem to still be active. So I'm hoping to understand:
1. What is bad about the Disconnect entities.json list? Is it how the list is curated? Something about how it conveys information?
2. The answer to (1) may answer this, but which issues of those does RWS still share in your mind?
3. Is Firefox still planning on reducing the ways it uses entities.json?

> We want the promise we make to be very clear: if the address bar changes, that's a new space.
Would this mean that the only way a 3P embed could see information across top level sites would be either something like FedCM, or an explicit request via something like SAA?

Reply:
There are a number of problems with lists, which is why we still hope to wean ourselves from a dependence on them.  The obvious problems are simple maintenance and ensuring they are correct.  The less immediate problems are that they often embed bias and inequity (I don't know how well Disconnect is able to serve sites from communities that exclusively use different languages, for example).  The pragmatic problem is that it takes a bunch of effort to keep lists synchronized with every browser instance.